governance-exercises/VoteWizard_poll_exercise_02/Derecognition of Clive AD Due to Compromised Operational Security - July 28, 2023.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Derecognition of Clive AD Due to Compromised Operational Security - JULY 28, 2023
+summary: Signal your support or opposition to derecognition of Clive AD due to compromised operational security. A community member, Diane, posted materials claiming that they could reveal the delegate's identity.
+discussion_link: https://forum.makerdao.com/t/...
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-28T16:00:00
+end_date: 2023-07-31T16:00:00
+---
+# Poll: Derecognition of Clive AD due to compromised operational security - JULY 28, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) has been posted as per [MIP106c2.6.6.2.1A](https://mips.makerdao.com/mips/details/MIP106#6-6-current-incubating-ecosystem-actors) and will be active for three days beginning on Monday, July 28 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the potential derecognition of Clive AD due to compromised operational security.
```

Comment:
It's worth being kind of clear around what's happening here. community is voting on whether these materials constitute significant suspicion which leads to the Facilitator being required to offboard given MIP101.

Arguably the facilitator shouldn't be polling on this at all, but given the exercise, it was assumed that they were doing so.

Reply:
Probably don't want 'potential' in there either.